Lotto/Lotto/Lotto/LottoInputView.swift

```diff
@@ -18,6 +18,21 @@ struct LottoInputView {
         return read(with: "구매금액을 입력해 주세요.")
     }
     
+    static func readNumberOfManualPurchase() -> Int {
```

Comment:
```suggestion
    static func readManualPurchaseCount -> Int {
```
는 어떨까 하고 저는 생각해보았습니다 😊